src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/mathfuncs.txt

```diff
@@ -139,6 +139,18 @@ Blank()
 >> Max(Blank(), 100, -5, "2000", Value("2001"), Text(2002))
 2002
 
+>> Max(DateTimeValue("November 10, 2014 1:50 PM"), DateTimeValue("November 11, 2014 1:50 PM"))
+"11/11/2014 1:50 PM"
+
+>> Max(DateTimeValue("November 10, 2014 1:50 PM"), DateTimeValue("November 10, 2014 2:50 PM"))
+"11/10/2014 2:50 PM"
+
+>> Min(Date(1985,3,5),Date(1986,10,19))
+"3/5/1985"
+
+>> Max(Time(1,50,0),Time(2,0,0))
+"1/1/1970 2:00 AM"
```

Comment:
This should be a time value, not a date/time.
![image](https://user-images.githubusercontent.com/44986255/155421714-1339da01-9538-4c25-9c2b-94fbd769f526.png)